code/datums/supplypacks/robotics.dm

```diff
@@ -108,6 +140,14 @@
 	containername = "Robolimb blueprints (Bishop)"
 	access = access_robotics
 
+/datum/supply_packs/robotics/robolimbs/rook
+	name = "Rook robolimb blueprints"
+	contains = list(/obj/item/weapon/disk/limb/rook)
+	cost = 55
```

Comment:
These prices are all going to need to be discussed.

Reply:
I'll make it a consistent *0.8 rounded up from the company they're a knockoff from.